docs/tutorials/basics.ipynb

```diff
@@ -68,7 +68,7 @@
     "id": "67gUU740MSXd"
```

Comment:
<p>More uses of the word optimizer in this paragraph that should be replaced by transformers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Done.</p>